test/service/test_portal_service.py

```diff
@@ -0,0 +1,169 @@
+import json
+import unittest
+
+import boto3
+from botocore.exceptions import ClientError
+from moto import (
+    mock_sts,
+    mock_s3,
+    mock_dynamodb2,
+)
+
+from azul import config
+from azul.logging import configure_test_logging
+from azul.plugin import Plugin
+from azul.portal_service import PortalService
+from azul.types import JSONs
+from azul.version_service import NoSuchObjectVersion
+from azul_test_case import AzulTestCase
+from service.test_version_service import create_version_table
```

Comment:
We typically share fixtures or other test functionality via inheritance. I would create a base class that both TestPortalService and TestVersionService inherit and put all the shared functionality there. This would also allow you to share the boto3 DynamoDB client object. I would call that class `VersionTableTestCase`.